### PR TITLE
Go version

### PR DESCRIPTION
**Pull request by AaronVasquez**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		}
+
+		// Get configuration.
+		provider := os.Getenv("PROVIDER")
+		minDepth, _ := strconv.ParseInt(os.Getenv("MIN_DEPTH"), 10, 64)
+		minWeightMag, _ := strconv.ParseInt(os.Getenv("MIN_WEIGHT_MAGNITUDE"), 10, 64)
+
+		// Async sendTrytes
+		api := giota.NewAPI(provider, nil)
+		_, pow := giota.GetBestPoW()
```
Comment (rfornea) [inline main.go:79]: I don't know much about Go.  What does that '_, ' do?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		http.Error(w, err.Error(), 500)
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Write(body)
+}
+
+func getFuncName(i interface{}) string {
+	return runtime.FuncForPC(reflect.ValueOf(i).Pointer()).Name()
+}
```
Comment (rfornea) [inline main.go:110]: Do the methods in this file execute broadcastTransactions and storeTransactions "behind the scenes"?

File: main.go (hunk 1 above)
Comment (AaronVasquez) [reply main.go:79]: Signals to the compiler that the variable is unused. In this case, the first return value is an error, which we just ignore.

File: main.go (hunk 2 above)
Comment (AaronVasquez) [reply main.go:110]: Yes, I believe so: https://godoc.org/github.com/iotaledger/iota.lib.go#SendTrytes.

```SendTrytes does attachToTangle and finally, it broadcasts the transactions.```

Tested an upload, and the client was able to download the file.

**Review by rfornea — APPROVED** [review]

**Pull request merged**